changelogs/react-hotfix-tabcontext-export.md

```diff
@@ -0,0 +1,3 @@
+Patch
+Fixed
+- (React) [TabContext] hotfix: Fixes export of TabContext in react package build. #616
```

Comment:
i would directly just update the changelog since its a hotfix so it can go into master